manim/utils/cfg_subcmds.py

```diff
@@ -0,0 +1,187 @@
+                            console.print(f"Enter the style for {key}:", style=key, end="")
+                            temp = input()
+                        else:
+                            default[key] = temp
+                default = replace_keys(default)
+
+            else:
+                for key in default:
+                    if default[key] in ["True","False"]:
+                        console.print(
+                            f"Enter value for {key} (defaults to {default[key]}):", end="")
+                        temp = input()
+                        if temp:
+                            while not temp.lower().capitalize() in ["True","False"]:
+                                console.print(
+                                    "[red bold]Invalid value. Try again.[/red bold]")
+                                console.print(
+                                    f"Enter the style for {key}:", style=key, end="")
+                                temp = input()
+                            else:
+                                default[key] = temp
+            config[category] = dict(default)
+    else:
+        action = "open"
+
+    if level is None:
+        console.print(
+            f"Do you want to {action} the default config for this User?(y/n)[[n]]",
+            style="dim purple",
+            end="",
+        )
+        action_to_userpath = input()
+    else:
+        action_to_userpath = ""
+
+    if action_to_userpath.lower() == "y" or level=="user":
+        cfg_file_path = os.path.join(
+            guarantee_existence(
+                os.path.dirname(config_paths[1])
+                ),"manim.cfg")
+        console.print(USER_CONFIG_MSG)
+    else:
+        cfg_file_path = os.path.join(
+            guarantee_existence(
+                os.path.dirname(config_paths[2])
+                ),"manim.cfg")
+        console.print(CWD_CONFIG_MSG)
+    with open(cfg_file_path, "w") as fp:
+        config.write(fp)
+    if openfile:
+        open_file(cfg_file_path)
+
+def show():
+    current_config = finalized_configs_dict()
+    for category in current_config:
+        console.print(f"{category}",style="bold green underline")
+        for entry in current_config[category]:
+            if category=="logger":
+                console.print(f"{entry} :",end="")
+                console.print(
+                    f" {current_config[category][entry]}",
+                    style=current_config[category][entry]
+                    )
+            else:
+                console.print(f"{entry} : {current_config[category][entry]}")
+        console.print("\n")
+
+def export(path):
+    config = _run_config()[1]
+    if os.path.abspath(path) == os.path.abspath(os.getcwd()):
+        console.print(
+            """You are reading the config from the same directory you are exporting to.
+This means that the exported config will overwrite the config for this directory.
+Are you sure you want to continue?[y/n]""",
```

Comment:
**Issue :** for a strange reason the `y/n` did not appear on my command prompt when it was between two `[` (see image)
![image](https://user-images.githubusercontent.com/36239975/88716647-a9222500-d11f-11ea-909a-ebeb859bdf39.png)

I fixed it by changing `[y/n]` by `(y/n)` : 
![image](https://user-images.githubusercontent.com/36239975/88716734-ca831100-d11f-11ea-8b75-dc9a41a2e8d5.png)
```suggestion
Are you sure you want to continue? (y/n)""",
```

Reply:
Ohh I think this is because Rich considers anything within square brackets to be a style, and doesn't treat it as _text_.

Pinging @naveen521kk to see if this is true.